### PR TITLE
Fixed compilation on rust 1.35.0

### PR DESCRIPTION
**Pull request by ko1N**
The current version of cargo-eval does not compile anymore with the following error:

```
error[E0277]: the trait bound `std::string::String: std::convert::From<&std::string::String>` is not satisfied
   --> src/main.rs:247:28
    |
247 |             content = expr.into();
    |                            ^^^^ the trait `std::convert::From<&std::string::String>` is not implemented for `std::string::String`
    |
    = help: the following implementations were found:
              <std::string::String as std::convert::From<&str>>
              <std::string::String as std::convert::From<std::borrow::Cow<'a, str>>>
              <std::string::String as std::convert::From<std::boxed::Box<str>>>
    = note: required because of the requirements on the impl of `std::convert::Into<std::string::String>` for `&std::string::String`

error[E0277]: the trait bound `std::string::String: std::convert::From<&std::string::String>` is not satisfied
   --> src/main.rs:251:29
    |
251 |             content = loop_.into();
    |                             ^^^^ the trait `std::convert::From<&std::string::String>` is not implemented for `std::string::String`
    |
    = help: the following implementations were found:
              <std::string::String as std::convert::From<&str>>
              <std::string::String as std::convert::From<std::borrow::Cow<'a, str>>>
              <std::string::String as std::convert::From<std::boxed::Box<str>>>
    = note: required because of the requirements on the impl of `std::convert::Into<std::string::String>` for `&std::string::String`
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The following features are defined:
 <a name="shebang"></a>
 ### Self-Executing Scripts
 
-On UNIX systems, you can use `#!/usr/bin/env cargo eval --` as a shebang line in a Rust script.  If the script file is executable, this will allow you to execute a script file directly.
+On UNIX systems, you can use `#!/usr/bin/env -S cargo eval --` as a shebang line in a Rust script.  If the script file is executable, this will allow you to execute a script file directly.
```
Comment (reitermarkus) [inline README.md:56]: Why is the `-S` flag necessary?

Comment (ko1N) [reply README.md:56]: I just checked and it only seems necessary on certain linux distros (in my case arch) to feed multiple arguments into /usr/bin/env. We could revert that change or just add a disclaimer that the -S flag might be needed.

Comment (reitermarkus) [reply README.md:56]: So Linux will see `env 'cargo eval --'` and macOS will see `env cargo eval --`. `env -S cargo eval --`  will still work on macOS, so let's leave it in.

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ env:
   - RUSTFLAGS='-D warnings'
 
 rust:
+  - 1.35.0
   - 1.36.0
```
Comment (reitermarkus) [inline .travis.yml]: Might as well bump this to 1.37 now.

Comment (reitermarkus) [reply .travis.yml]: Or remove it, since we already have `stable`.

Comment (ko1N) [reply .travis.yml]: Ye, building against stable and nightly should be enough. I just added 1.35.0 to check for backwards compatability. Do you have any idea why the windows builds fail in travis?

Comment (reitermarkus) [reply .travis.yml]: Actually I just meant remove 1.36 and only keep 1.35. No idea why the windows builds are failing. They are generally very unreliable, I'll move this to Azure Pipelines when I get the chance.

Comment (ko1N) [reply .travis.yml]: Maybe going back to the oldest rust version u intend to support would be a good choice for the CI (despite using stable/latest).

Comment (reitermarkus) [reply .travis.yml]: I didn't intend to support anything older than 1.36, but your fix is simple enough, so going back to 1.35 is fine.

**Comment by reitermarkus** [comment]
Merged in https://github.com/reitermarkus/cargo-eval/commit/265c63c01432303c9596e0e042b303cf5308d849.

**Pull request closed without merging**